ha/core/controllers/pcs/cluster_controller.py

```diff
@@ -64,6 +66,19 @@ def _pcs_cluster_start(self):
             Log.info(f"Pcs cluster start retry index : {retry_index}")
         return _status
 
+    def _auth_node(self, node_id, cluster_user, cluster_password):
```

Comment:
This method will need to be called from ha_setup.py for adding additional nodes too. So it should be exposed,rt?

Reply:
Add node is also part cluster manager which internally doing auth node
So, setup only call add_node and create_cluster, auth happens internally

Reply:
sure